plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerPage.tsx

```diff
@@ -37,7 +37,7 @@ export const ApiExplorerPage = () => {
             variant="contained"
             color="primary"
             component={RouterLink}
-            to="/register-component"
+            to="/catalog-import"
```

Comment:
There may be a more clean way to dynamically infer the route now (saw additions being added I think to let them be discovered more cleanly?), but for now kept with the hardcoded one like was already there